src/reflect/type.go

```diff
@@ -850,6 +918,8 @@ func (t *rawType) Size() uintptr {
 	}
 }
 
+var structAlignCache = make(map[*rawType]int)
```

Comment:
We could probably added additional caches for `Size()`, `Comparable()`, and `isBinary()`. (assuming we didn't want to add Comparable and isBinary to a flag byte somewhere already)